### PR TITLE
docs(/contact-us): add Discord server invite link

### LINKED ISSUE #2023: Add Discord link to /contact-us
Our Discord community is growing and adding a link to the /contact-us page makes sense 🏄

### PR DESCRIPTION
**Pull request by dakahn**
Closes #2023 

Adds a Discord server invite link to the /contact-us page. Worth noting is that it's _above_ our Slack links which led off the section previously. Made sense to me to have our first point of contact reachable as an open source project, but happy to adjust if I need to.

### CODE REVIEW

**Review by shinytoyrobots — APPROVED** [review]

File: src/pages/help/contact-us/index.mdx [hunk 1]
```diff
@@ -70,6 +70,11 @@ the appropriate Carbon repo.
 
 ## Start a new discussion
 
+### Discord
+
+Come chat and get support from Carbon Maintainers and fellow community members
```
Comment (janchild) [inline src/pages/help/contact-us/index.mdx:75]:
```suggestion
Come chat and get support from Carbon maintainers and fellow community members
```

File: src/pages/help/contact-us/index.mdx [hunk 2]
```diff
@@ -70,6 +70,11 @@ the appropriate Carbon repo.
...
+on our [Discord Server](https://discord.gg/SYjvP8epkM).
```
Comment (janchild) [inline src/pages/help/contact-us/index.mdx:76]:
```suggestion
on our [Discord server](https://discord.gg/SYjvP8epkM).
```

**Review by janchild — COMMENTED** [review]
@dakahn just a couple of little capitalization fixes. 

Thanks!
Jan

**Review by tw15egan — APPROVED** [review]

**Review by janchild — APPROVED** [review]

**Comment by dakahn** [comment]
At a loss on this one here. Cleared cache, scrapped yarn lock, triple checked my Node version and reinstalled 🤷

**Comment by tw15egan** [comment]
@dakahn want to just try a new PR? Not sure what the ~20,000 `yarn.lock` changes are

**Review by emyarod — COMMENTED** [review]
the deletion of the lockfile is committed in this PR. you can just undo that deletion so only the changes to the MDX file remain

**Pull request closed without merging**